src/ui/Main.tsx

```diff
@@ -59,6 +62,8 @@ export class Main extends React.Component<IProps, IState> {
         alert('Error launching game')
       },
       notification: null,
+      isGamePathSet: false,
```

Comment:
```suggestion
      isGamePathSet: true,
```

Reply:
This prevents notification every time launcher is opened, like Spike mentioned before. Notification will still properly occur when path is not set or incorrect, but will stop it from flashing on open when the path is set correctly.